index.html

```diff
@@ -33,9 +36,149 @@
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+       
+            <ul class="tagcontainer">
+         <li class="label-project">Ruby on Rails</li>
+         <li class="label-project">Javascript</li>
+         <li class="label-project">CSS</li>
+         <li class="label-project">HTML</li>
+            </ul>
+        
+        <button class="see-post"><p class="see-button">see project</p></button></div>
+        <div class="grid-item sixth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+          
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+   
+        <div class="about-me">
+            
+            <img class="right" src="./right.png" alt="image">  
+            <h2 class="about">About Me</h2>
+            
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+               
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
+               <img class="left" src="./left.png" alt="image"> 
+        </div>
+         <div class="grid-container-another">
+        <div class="grid-item-another">
+            <img class="skills languagelogo" src="./icon-languages@2x.png" alt="language logo">
+            <h3 class="language">Languages</h3>
+            
+                <ul class="mini-flex-container-one">
+                <li class="one mini-item"><p>Javascript</p></li>
+                 <li class="two mini-item"><p>HTML</p></li>
+                 <li class="three mini-item"><p>Ruby</p></li> 
+                 <li class=" four mini-item"><p>CSS</p></li> 
+                 </ul> 
```

Comment:
- [ ] kindly fix this design flaw which is currently on your web page so that it matches the figma design below 
> - figma design
![ff-s](https://user-images.githubusercontent.com/89055502/235324527-1ddb5b71-a245-472c-b6cd-f1dcf9a79666.png)
> - your design
![f-ss](https://user-images.githubusercontent.com/89055502/235324546-e37912f0-70ef-4351-b1a2-2c35fa4bb9e7.png)